Frontend/ui-customer/src/Flow.purs

```diff
@@ -2500,6 +2535,69 @@ homeScreenFlow = do
       else do
         modifyScreenState $ PickupInstructionsScreenStateType $ \pickupInstructionsScreen -> pickupInstructionsScreen { data { pickupLat = lat, pickupLong = lon, pickupInstructions = pickupInstructions } }
         pickupInstructionsScreenFlow
+    UPDATE_DROP_NAME state lat lon -> do 
+        (ServiceabilityRes destServiceabilityResp) <- Remote.locServiceabilityBT (Remote.makeServiceabilityReq lat lon) DESTINATION
```

Comment:
@dhruv-1010 can we combine this flow with already existing flow :  UPDATE_PICKUP_NAME

Reply:
same like above. sent some tag. based on that modify the flow

Reply:
sure working on it